src/models/user/stream.rs

```diff
@@ -1,47 +1,63 @@
-use super::{Muted, UserCounts, UserSearch, UserView};
+use super::{Influencers, Muted, UserCounts, UserSearch, UserView};
 use crate::models::follow::{Followers, Following, Friends, UserFollows};
-use crate::types::DynError;
+use crate::types::{DynError, StreamReach, Timeframe};
 use crate::{db::kv::index::sorted_sets::SortOrder, RedisOps};
 use crate::{get_neo4j_graph, queries};
 use serde::{Deserialize, Serialize};
 use tokio::task::spawn;
 use utoipa::ToSchema;
 
 pub const USER_MOSTFOLLOWED_KEY_PARTS: [&str; 2] = ["Users", "MostFollowed"];
-pub const USER_PIONEERS_KEY_PARTS: [&str; 2] = ["Users", "Pioneers"];
+pub const USER_INFLUENCERS_KEY_PARTS: [&str; 2] = ["Users", "Influencers"];
```

Comment:
This is a breaking change of Index schemas and we do not have a "migration". What is the plant? Is this rename needed just yet? If we update or staging sever the feature will stop working.

Reply:
this was fixed like this because the graph is still too small for a reindex to be costly